java/aiservices/openai/src/main/java/com/microsoft/semantickernel/aiservices/openai/implementation/OpenAIRequestSettings.java

```diff
@@ -53,6 +55,9 @@ private static String loadVersion() {
      */
     public static RequestOptions getRequestOptions() {
         return new RequestOptions()
-            .setHeader(HttpHeaderName.fromString("Semantic-Kernel-Version"), version);
+            .setHeader(HttpHeaderName.fromString("Semantic-Kernel-Version"), "Java/" + version)
```

Comment:
For this header, how about aligning with Pyhton?

`python-<version>` is the format they are using, I believe.